_integration-tests/tests/sql/sql.go

```diff
@@ -25,6 +26,9 @@ func (tc *TestCase) Setup(t *testing.T) {
 	var err error
 	tc.DB, err = sql.Open("sqlite3", "file::memory:")
 	require.NoError(t, err)
+	t.Cleanup(func() {
+		assert.NoError(t, tc.DB.Close())
```

Comment:
Why replace require by the assert package ?

Reply:
I think we don;t need to use `require` in the cleanup function - as I mentioned in the previous comment, the test (if there is anything remaining) can continue despite of this failing.